openslides/agenda/models.py

```diff
@@ -372,14 +380,14 @@ class Speaker(RESTModelMixin, models.Model):
 
     objects = SpeakerManager()
 
-    user = models.ForeignKey(User)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL)
```

Comment:
Are you absolutely sure you can use the setting? This can be a string. See https://docs.djangoproject.com/en/1.8/topics/auth/customizing/#substituting-a-custom-user-model